html-css/week1/nadiia/portfolio/styles.css

```
@@ -0,0 +1,1985 @@
+  border-radius: 50px;
+  object-fit: cover;
+  aspect-ratio: 1/1;
+  max-width: 100%;
+}
+.cats {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 900px;
+  margin-top: 150px;
+}
+.catblackleft {
+  bottom: 0;
+  display: block;
+  width: 200px;
+  height: 200px;
+}
+.catblackright {
+  bottom: 0;
+  display: block;
+}
+.catblackhead {
+  position: relative;
+  width: 129px;
+  height: 129px;
+  background-color: #252020;
+  bottom: 100px;
+  border-radius: 50%;
+}
+.catblackeye {
+  width: 16px;
+  height: 16px;
+  bottom: 80px;
+  background-color: #f1aac7;
+  border-radius: 50%;
+}
+.blacklefteye {
+  position: relative;
+  left: 80px;
+  top: 50px;
+}
+.blackrighteye {
+  position: relative;
+  left: 30px;
+  top: 30px;
+}
+.catblacknose {
+  width: 0;
+  height: 0;
+  border-top: 5px solid #f1aac7;
+  border-left: 10px solid transparent;
+  border-right: 10px solid transparent;
+  position: relative;
+  left: 48px;
+  transform: rotate(5deg);
+  top: 30px;
+}
+.catblackears {
+  width: 0;
+  height: 0;
+  border-bottom: 40px solid #252020;
+  border-left: 35px solid transparent;
+  border-right: 35px solid transparent;
+}
+.blackleftear {
+  position: relative;
+  top: -60px;
+  left: 5px;
+  transform: rotate(-20deg);
+}
+.blackrightear {
+  position: relative;
+  top: -80px;
+  left: 85px;
+  transform: rotate(48deg);
+}
+footer {
+  background-color: #252020;
+  color: #fbf5e5;
+  padding-top: 100px;
+  padding-bottom: 100px;
+}
+#contact {
+  border: 5px solid #c8648f;
+  border-radius: 50px;
+  margin: 0 auto;
+  width: 75%;
+}
+.contacttext {
+  text-align: center;
+  font-size: 32pt;
+  margin: 25px 0;
+}
+.icon {
+  width: 150px;
+  height: 150px;
+  border: none;
+  border-radius: 25px;
+}
+.icon:hover {
+  background-color: #3b3333;
+}
+.icon img {
+  border-radius: 25px;
+  border: 3px solid #c8648f;
+}
+.align {
+  display: flex;
+  flex-flow: row;
+  justify-content: center;
+  gap: 25px;
+  margin-bottom: 50px;
+}
+.plus {
+  margin: 0 10%;
+}
+li {
+  list-style-type: none;
+}
+
+/*media query works on android tablet and smartphones, but it doesn't work on IPhone for some reason. Maybe the issues happen only in IPhone5*/
+@media (max-width: 1020px) {
+  .catright {
+    display: none;
+  }
+  .undertext {
+    font-size: 18pt;
+    font-weight: bold;
+  }
+  .art_project {
+    font-size: 28pt;
+    padding-top: 50px;
+  }
+  #about {
+    padding-top: 200px;
+  }
+  .art_project_item {
+    max-width: 40%;
+  }
+  .project {
+    gap: 0;
+  }
+  .art_project_item_art {
+    max-width: 40%;
+  }
+  .catblackleft {
+    display: none;
+  }
+  .contacttext:first-of-type {
+    padding-top: 10px;
+  }
+}
+@media (max-width: 840px) {
+  nav {
+    justify-content: center;
+  }
+  a::after {
+    display: none;
+  }
+  .a1 {
+    animation: none;
+  }
+  .a2 {
+    animation: none;
+  }
+  .a3 {
+    animation: none;
+  }
+  .a4 {
+    animation: none;
+  }
+  .a5 {
+    animation: none;
+  }
```

You can combine selectors like this:
```suggestion
  .a1, .a2, .a3, .a4, .a5 {
    animation: none;
  }
```